widgets/contact/contact.php

```diff
@@ -343,6 +343,72 @@ function get_widget_form(){
 						),
 					),
 
+					'fields' => array(
+						'type'   => 'section',
+						'label'  => __( 'Fields', 'so-widgets-bundle' ),
+						'fields' => array(
+							'font' => array(
+								'type'    => 'font',
```

Comment:
To be honest, I forgot to do this. :(
I do like the idea of automating this however as it feels a tad redundant to me.